inspirehep/modules/workflows/tasks/merging.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Tasks related to record merging."""
+
+from __future__ import absolute_import, division, print_function
+from invenio_db import db
+from inspirehep.modules.workflows.models import WorkflowsRecordSources
+
+
+def read_wf_record_source(record_uuid, source):
+    entry = WorkflowsRecordSources.query.filter_by(
+        record_id=str(record_uuid),
+        source=source.lower()
+    ).one_or_none()
+    return entry
```

Comment:
A function that does exactly one function calls and returns smells of ravioli. I see that you are using it two times here and several times in the tests: perhaps it's time to graduate it to an official util, since, in essence, this function is the workflow analogue of `get_db_record`.

Reply:
I created ``workflow.py`` in utils, wdyt?

Reply:
moved to ``workflow/utils``.

Reply:
moved to ``workflows/utils``.